packages/shopify-codemod/test/fixtures/coffeescript-soak-to-condition/assignment.output.js

```diff
@@ -0,0 +1,16 @@
+if (typeof bar === 'function' && bar() != null && bar().baz.qux != null) {
```

Comment:
I feel very strongly that we shouldn't change the effect of code, we could introduce some really insidious bugs this way